src/pages/Hyun Yong Jeong/Signup/Signup.scss

```
@@ -0,0 +1,99 @@
+@mixin button {
+  width: 100%;
+  color: white;
+  font-size: 16px;
+  background-color: rgb(94, 94, 250);
+  margin-bottom: 0px;
+  border-radius: 5px;
+  border: 1px solid rgb(210, 197, 197);
+  padding: 18px 12px;
+}
+
+@mixin includeText {
+  font-size: 16px;
+  margin-bottom: 10px;
+  border-radius: 5px;
+  border: 1px solid rgb(228, 212, 212);
+  padding: 18px 12px;
+}
+
+.info {
+  width: 100%;
+  margin: 0px auto;
+  margin-top: 60px;
+  margin-bottom: 30px;
+}
+
+input {
+  width: 100%;
+  @include includeText;
+}
+input::placeholder {
+  color: rgb(204, 185, 185);
+}
```

&선택자를 활용하면 좀  더 간단하게 작성할 수 있습니다.
```suggestion
input {
  width: 100%;
  @include includeText;
  &::placeholder{
  color: rgb(204, 185, 185);
  }
}
```